charts/README.md

```diff
@@ -1,6 +1,6 @@
 # Meshery Chart
 
-Meshery is the multi-service mesh management plane. This repository include meshery Helm chart.
+Meshery is the Cloud Native management plane. This repository include meshery Helm chart.
```

Comment:
Lowercase, please.

Reply:
will do. Also, shouldn't it be "This repository includes the Meshery Helm chart." ? @leecalcote